distributed-job-manager/src/main/java/ru/fix/distributed/job/manager/WorkPoolRunningStrategies.java

```diff
@@ -15,7 +15,7 @@ private WorkPoolRunningStrategies() {
 
     @NotNull
     public static WorkPoolRunningStrategy getSingleThreadStrategy() {
-        return localWorkPool -> 1;
+        return localWorkPool -> localWorkPool.isEmpty()? 0 : 1;
```

Comment:
Could you please provide test when 
1. we start DJM with single job that has empty work pool
2. work pool of the job is changed to non-empty
Expecting:  
- job is not launched, from metrics  we see that job did not started and we see that there is empty workpool for it
- number of threads changed correctly, job is launched, metrics are recorded

That kind of test and metrics will ensure that we can detect reason that job did not started due to empty workPool and also can be sure that 0 number of threads does not kill our scheduller.

Reply:
`threadsCount = Math.max(threadsCount, 1);` should save us, but better to have a test case for that.